pkgs/applications/audio/pd-plugins/maxlib/default.nix

```diff
@@ -1,28 +1,26 @@
-{ stdenv, fetchurl, puredata }:
+{ stdenv, fetchFromGitHub, puredata }:
 
 stdenv.mkDerivation rec {
   name = "maxlib-${version}";
-  version = "1.5.5";
+  version = "1.5.7";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/project/pure-data/libraries/maxlib/${name}.tar.gz";
-    sha256 = "0vxl9s815dnay5r0067rxsfh8f6jbk61f0nxrydzjydfycza7p1w";
+  src = fetchFromGitHub {
+    owner = "electrickery";
+    repo = "pd-maxlib";
+    rev = "v${version}";
+    sha256 = "10w9qfgn26lj3zqjksf2r1wsjpf5xy4dx22jay9l6idy9q62mxsn";
   };
```

Comment:
This does not look like the official continuation of maxlib. For instance, `meta.homepage` below does not link here, and the developer is different. We may replace the source, but could you explain why?

Reply:
Maybe I'm too gullable, but the readme.md of this source says:
``Please note that Olaf does not maintain this code any longer, so questions and bug-reports should be directed towards me.``
Furthermore it seems like he's only doing bugfixes.

Reply:
OK.